articles/data-preprocessing/index.md

```diff
@@ -0,0 +1,167 @@
+
+Data preprocessing is the initial phase of machine learning where data is prepared for machine learning models. This part is very crucial in machine learning and needs to be done properly and in a systematic manner. If not, we will end up building models that are not accurate for their purpose.
+
+In this article, we shall learn all steps that's involved in the data preprocessing stage. 
+
+### Prerequisites
+
+1. [RStudio](https://www.rstudio.com/products/rstudio/download/) install on your computer.
+
+2. install packages('caTools')
+
+### Steps in Data Preprocessing
+
```

Comment:
Please provide an overview of the steps here. Just the headings. If possible, add links to the respective sections in the article.